src/App.js

```diff
@@ -31,23 +34,42 @@ const useStyles = makeStyles(() => ({
   logo: {
     height: 60,
     width: 60,
+    marginTop: 10,
+    marginLeft: 10,
   },
 }));
 
+const StyledTabs = withStyles((theme) => ({
+  indicator: {
+    display: "flex",
+    justifyContent: "center",
+    backgroundColor: "transparent",
+    "& > span": {
+      maxWidth: 85,
+      width: "100%",
+      backgroundColor: theme.palette.primary.contrastText,
+    },
+    height: "4px",
+  },
+  root: {
+    float: "right",
+    paddingRight: "20px",
+    [theme.breakpoints.down("md")]: {
+      display: "none",
+    },
+  },
+}))((props) => <Tabs {...props} TabIndicatorProps={{ children: <span /> }} />);
+    
 const StyledTab = withStyles((theme) => ({
   root: {
     textTransform: "uppercase",
     fontWeight: "bold",
-    fontSize: "1.6vw",
+    fontSize: "25px",
     color: theme.color,
     marginRight: theme.spacing(1),
-
     "&:focus": {
       opacity: 1,
     },
-    "&:hover": {
-      textDecoration: "none",
-    },
```

Comment:
Why get rid of this? This prevents the text having an underline on hover when it's wrapped in React Router Link. Also need to prevent it from turning blue.

Probably also need to do this for the sidebar menu (it underlines atm on hover)

Reply:
Added these few lines back